test/simulator/main/org/apache/cassandra/simulator/systems/InterceptibleThreadFactory.java

```diff
@@ -58,7 +58,9 @@ public InterceptibleThread newThread(Runnable runnable)
         @Override
         protected synchronized InterceptibleThread newThread(ThreadGroup threadGroup, Runnable runnable, String name)
         {
-            InterceptibleThread thread = new InterceptibleThread(threadGroup, runnable, name, extraToStringInfo, onTermination, parent.interceptorOfGlobalMethods, time);
+            // Can not use NamedThreadFactory.globalPrefix() as this method runs in the App class loader and not the Instance class loader; the ThreadGroup's name can act as a proxy for this.
+            String threadName = threadGroup.getName() + '_' + name;
```

Comment:
so with Simulator doing threads itself it was missing the node prefix, which made the logs far harder to understand... so i keep the same semantics as normal jvm-dtest.

Reply:
Is there no way to have the logs include the thread group name, rather than replicating it?